ui/inspectors/system-section.reel/updates/updates.reel/updates.js

```diff
@@ -71,7 +71,12 @@ exports.Updates = Component.specialize(/** @lends Updates# */ {
 
     handleCheckDownloadAction: {
         value: function() {
-            this._updateService.checkAndDownload();
+            var self = this;
+            this._updateService.checkAndDownload().then(function() {
```

Comment:
`getInfo` is used separately in `enterDocument` for `updates.js`. I think it should stay that way.